scripts/local/build-image_test.sh

```
@@ -73,5 +73,42 @@ else
+# "existing blob" and never crosses the wire. Both halves in one tar throws that
+# away, and it throws it away SILENTLY — the publish succeeds, the image is
+# correct, and the only symptom is that it took twice as long.
+#
+# Measured on the 2026-09-05 publish, before the split: seeding 679 glossary rows
+# grew 3gpp.duckdb by 9.7 MB and left etsi.duckdb identical to the byte, and
+# 19.7 GB of unchanged ETSI went up the wire regardless. Nothing failed. Nothing
+# reported it. That is why this is pinned in a test rather than a comment.
+#
+# Read out of the real script, not restated here: a copy would keep passing after
+# the original was changed back.
+corpus_layers="$(grep -E '^\[ "\$WITH_CORPUS" = 1 \] && layer ' "$SCRIPT")"
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)